e2e-tests/web/test/specs/navigateToDashboard.e2e.ts

```diff
@@ -0,0 +1,33 @@
+import LandingPage from '../pageobjects/landing.page'
+
+describe('navigate to dashboard', () => {
+  it('Filter and Navigate to City', async () => {
+    // given
+    const language = 'en'
+    const filter = 'Augsburg'
+    const dashboardPath = `/${filter.toLowerCase()}/en`
+    await browser.setTimeout({ implicit: 80000 })
```

Comment:
Why do we need this timeout? Should we set this globally?

Reply:
I moved it into the config. It specifies how long we wait for location strategies such as findElement, $, $$, ...